website/content/docs/provisioners/ansible.mdx

```diff
@@ -34,6 +34,27 @@ DigitalOcean. Replace the mock `api_token` value with your own.
 Example Packer template:
 
 <Tabs>
+<Tab heading="HCL2">
```

Comment:
yup yup ! https://packer-38ptvnalf-hashicorp.vercel.app/docs/provisioners/ansible#basic-example

Reply:
fixed formatting on this